feat(tag): add query help documentation (#23)
ni/systemlink-grafana-plugins: src/datasources/tag/query_help.md

### SystemLink Tags

> This data source allows you to visualize both the current and historical values

Reviewer: ```suggestion
This data source allows you to visualize the current and historical values
```

of tags.

#### Query fields

> **Query type** - Choose **Current** to visualize the tag's most recent value, or

Reviewer: ```suggestion
**Query type** - **Current** allows you to visualize the most recent tag value.
```

> **History** to visualize its values over time. Historical values will be

Reviewer: ```suggestion
**History** allows you to visualize tag values over time. Historical values use
```

> retrieved within the time range set on the dashboard and decimated according to

Reviewer: ```suggestion
the time range set on the dashboard and are decimated according to
```

the width of the panel.

> **Tag path** - The full path of the tag you want to visualize. You can input a

Reviewer: ```suggestion
**Tag path** - The full path of the tag you want to visualize. You can enter a
```

[variable](https://grafana.com/docs/grafana/latest/dashboards/variables/) into
this field.

> **Workspace** - The id of the workspace in which to search for the given tag

Reviewer: ```suggestion
**Workspace** - The ID of the workspace to search for the given tag path.
```

> path. If left blank, the plugin finds the most recently updated tag in any

Reviewer: ```suggestion
If left blank, the plugin finds the most recently updated tag in any workspace.
```
